schema/filter/impl/Th2MsgFieldExtraction.go

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2023 Exactpro (Exactpro Systems Limited)
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package filter
+
+import (
+	p_buff "th2-grpc/th2_grpc_common"
+)
+
+type th2MsgFieldExtraction struct{}
+
+const SESSION_ALIAS_KEY = "session_alias"
+const MESSAGE_TYPE_KEY = "message_type"
+const DIRECTION_KEY = "direction"
+const PROTOCOL_KEY = "protocol"
```

Comment:
You can write `const` once and then define all constants you need
```suggestion
const (
 SESSION_ALIAS_KEY = "session_alias"
 MESSAGE_TYPE_KEY = "message_type"
 DIRECTION_KEY = "direction"
 PROTOCOL_KEY = "protocol"
 )
```